models/ContentShareQuery.php

```diff
@@ -17,23 +17,4 @@ public function forPublishing()
         return $this->andWhere(['status_id' => [ContentShare::STATUS_NEW, ContentShare::STATUS_FAILED]]);
     }
 
-    /**
-     * @inheritdoc
-     *
-     * @return ContentShare[]|array
-     */
-    public function all($db = null)
```

Comment:
Why?

Reply:
it's just calling the parent method, why is it needed?

Reply:
is it for IDE completion?

Reply:
It is.

Reply:
Wouldn't a PHPDoc comment at the class level be better?
```
 * @method ContentShare[]|array all($db = null)
```

Reply:
Likely.

Reply:
Done in https://github.com/yiisoft-contrib/yiiframework.com/pull/428/commits/9cb834e4823fd114b2083e05c197178b61d5a28c